### PR TITLE
curvefs/client: fix waitIntervalSec_ is not init

### LINKED ISSUE #1226: s3 adaptor unit test optimization
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**
The current s3 adaptor only has integration tests, no complete unit tests, and needs to be supplemented

**Describe the solution you'd like (描述你期望的解决方法)**

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by baijiaruo**
### What problem does this PR solve?

Issue Number: close #1226  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by baijiaruo** [comment]
recheck

**Comment by baijiaruo** [comment]
recheck

**Comment by baijiaruo** [comment]
recheck

**Comment by baijiaruo** [comment]
recheck

File: curvefs/src/client/s3/client_s3_adaptor.cpp [hunk 1]
```diff
@@ -61,6 +61,7 @@ S3ClientAdaptorImpl::Init(
     inodeManager_ = inodeManager;
     mdsClient_ = mdsClient;
     fsCacheManager_ = fsCacheManager;
+    waitIntervalSec_.Init(option.intervalSec * 1000);
```
Comment (ilixiaocui) [inline curvefs/src/client/s3/client_s3_adaptor.cpp:64]: why waitIntervalSec_ need option.intervalSec * 1000?

**Review by wu-hanqing — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**